graph/src/blockchain/mock.rs

```diff
@@ -268,6 +272,19 @@ impl<C: Blockchain> RuntimeAdapter<C> for MockRuntimeAdapter {
     }
 }
 
+pub struct MockBlockIngestor {}
```

Comment:
Is this actually used anywhere? Couldn't find it with a grep through the sources

Reply:
you're right, I had added this as a type on the trait but then removed it. Will remove this as well